library/dhm.c

```diff
@@ -524,14 +524,10 @@ int mbedtls_dhm_parse_dhm( mbedtls_dhm_context *dhm, const unsigned char *dhmin,
 #if defined(MBEDTLS_PEM_PARSE_C)
     mbedtls_pem_init( &pem );
 
```

Comment:
Need to update the docs in dhm.h, which refer to buflen needing to allow for a trailing NULL.

Reply:
Same